### ISSUE TITLE
Validation set size

### ISSUE THREAD

**Issue by ybendou** [linzhiqiu/cross_modal_adaptation#4]
Hi, 

Thank you for the very interesting repository and congrats for the paper! 
I have a question regarding the number of images on validation set for the hyperparameters selection. How many images of class do you use? 
Thank you in advance,

Best regards,

**Comment by linzhiqiu** [comment]
Thanks for your interest in our work! We use max(num_train_shot, 4) for the validation set following the evaluation protocol of CoOp. This means that for 1- or 2-shot setup we use 1 or 2 validation images per class. For 4/8/16-shot setup, we use 4 -shot validation images. All train/val image shots are sampled randomly. We have released the sample indices that we use and it should be the same as CoOp's.
